rust/hyperlane-base/src/contract_sync/mod.rs

```diff
@@ -77,16 +84,22 @@ where
                         "Found log(s) in index range"
                     );
                     // Store deliveries
-                    let stored = self.db.store_logs(&logs).await?;
+                    let Ok(stored) = self.db.store_logs(&logs).await else {
```

Comment:
unsure if it's worth sleeping for these ones - I'm assuming this one and the one below _should_ be propagated